pkg/remote/remote.go

```diff
@@ -153,10 +154,12 @@ type Params struct {
 	// to a random value which essentially means no-cache. Setting this to a
 	// static or known value will reuse the build cache
 	CacheInvalidationKey string
-	TemplateName         string
-	DockerfileName       string
-	KnownHostsPath       string
-	BaseImage            string
+	// CacheNamespace is used to allow multiple builds to run concurrently without interfering with each other cache
+	CacheNamespace string
```

Comment:
Why `CacheNamespace`? should it be something like `CachebyName`?

Reply:
I don't mind `CacheByName`, it's very similar to `CacheNamespace` semantically, initially I was going for `Name` but it wasn't specific enough in the end we're adding it for grouping cache builds, so namespace made sense. Does anyone have other names they want to suggest? @jLopezbarb @maroshii

Reply:
I don't want the `namespace` part of `CacheNamespace` get confused with the `OKTETO_NAMESPACE` for instance, so I willl rename it to `CacheByName` for now

Reply:
changed in 3a982e1167924fb2b9e92d7f3c78c17710bbf8d0

Reply:
changed in ffca6730909cdb276e7a08e4cecbb5f7b9d3fa43